tools/keyvaluestore.go

```diff
@@ -0,0 +1,201 @@
+package tools
```

Comment:
I think this would be appropriate to put in a sub-package of `tools`: `github.com/git-lfs/git-lfs/tools/kv`, which would allow us to get rid of a lot of `KeyValue*` prefixes.